src/reyna.Facts/GivenAForwardService.cs

```diff
@@ -83,6 +102,44 @@ public void WhenCallingStartAndMessageAddedShouldCallPostOnHttpClient()
         }
 
         [Fact]
+        public void WhenCallingStartAndItIsBlackoutShouldNotCallPostOnHttpClientOrDeleteMessage()
+        {
+            var networkInterface = new NetworkInterface();
+            networkInterface.CurrentIpAddress = new IPAddress(42);
+            networkInterface.Name = "cellular line";
+            var from = new Time();
+            var to = new Time(from.MinuteOfDay + 1);
+            this.Preferences.SetCellularDataBlackout(new TimeRange(from, to));
```

Comment:
this might be a little fragile - if time is about to change hh:mm:ss:999 for example